prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -808,15 +818,6 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             )
         }
 
-        /**
-         * Max number of hours to wait for a Batch task to succeed.
-         * We only run batch on tasks created after the backstop time.
-         */
-        const val BACKSTOP_HOURS = 26L
```

Comment:
Yuck - my initial pass at this was a global fixed backstop time.   The new per-receiver way is better